src/backend/InvenTree/part/models.py

```diff
@@ -841,15 +884,13 @@ def validate_unique(self, exclude=None):
                     'IPN': _('Duplicate IPN not allowed in part settings')
                 })
 
-        # Ensure unique across (Name, revision, IPN) (as specified)
-        if (
-            Part.objects.exclude(pk=self.pk)
-            .filter(name=self.name, revision=self.revision, IPN=self.IPN)
-            .exists()
-        ):
-            raise ValidationError(
-                _('Part with this Name, IPN and Revision already exists.')
```

Comment:
C (blocking): This change in business rules is breaking and I would like to discuss if this should be a setting

Reply:
Good catch this was not an intentional removal. Have added it back in.